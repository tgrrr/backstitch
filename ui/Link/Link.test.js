import React from 'react'
import renderer from 'react-test-renderer'
import { Link } from '.'

it('Link: default', () => {
    const component = renderer.create(
        <Link>
            <>
                Link
            </>
        </Link>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
