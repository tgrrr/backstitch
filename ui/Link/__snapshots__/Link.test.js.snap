// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Link "
    data-testid="Link"
  >
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      onBlur={[Function]}
      onFocus={[Function]}
    >
      Link
    </a>
  </div>
</div>
`;
